### PR TITLE
[CHG] extract computation of the partner_path in order to introduce …

### PR DESCRIPTION
**Pull request by JonathanNEMRY**
…more modularity into the methods

### CODE REVIEW

File: mass_mailing_distribution_list/distribution_list.py [hunk 1]
```diff
@@ -321,6 +321,12 @@ def _register_hook(self, cr):
             mass_mailing_models.append(model[0])
         type(self).dst_model_id.domain = [('model', 'in', mass_mailing_models)]
 
+    def get_opt_res_ids(
```
Comment (Olivier-LAURENT) [inline mass_mailing_distribution_list/distribution_list.py]: A private method would be better

File: mass_mailing_distribution_list/distribution_list.py [hunk 2]
```diff
@@ -321,6 +321,12 @@ def _register_hook(self, cr):
...
+            self, cr, uid, model_name, domain, in_mode=True, context=None):
```
Comment (Olivier-LAURENT) [inline mass_mailing_distribution_list/distribution_list.py]: remove default value of `in_mode` arg., it is always provided

**Comment by Olivier-LAURENT** [comment]
:+1: (with tests)

**Comment by laetitia-gangloff** [comment]
:+1:

**Comment by Cedric-Pigeon** [comment]
:+1:

**Pull request merged**